### ISSUE TITLE
`@debug` does not print map entries when the key is `null`

### ISSUE THREAD

**Issue by chriseppstein** [sass/libsass#1331]
```scss
$m: (foo: 1px, null: 2px);
@debug $m;
@debug map-get($m, null);
```

Prints out:

```
stdin:1: DEBUG: (foo: 1px)
stdin:2: DEBUG: 2px
```

**Comment by chriseppstein** [comment]
More fun stuff about `@debug`:

```scss
$m: (foo: 1px, null: 2px, false: 3px, true: 4px);
@debug $m;
@debug map-get($m, null);
@debug map-get($m, false);
@debug null;
```

prints out:

```
stdin:1: DEBUG: (foo: 1px, true: 4px)
stdin:2: DEBUG: 2px
stdin:3: DEBUG:
stdin:4: DEBUG:
```

Should print out:

```
Line 2 DEBUG: (foo: 1px, null: 2px, false: 3px, true: 4px)
Line 3 DEBUG: 2px
Line 4 DEBUG: 3px
Line 5 DEBUG: null
```

**Comment by xzyfer** [comment]
Thanks @chriseppstein this should be a straight forward fix.

**Comment by xzyfer** [comment]
As an aside I've noticed the following now produces an error

```scss
$m: (foo: 1px, null: 2px, false: 3px, true: 4px);
```
```
Error: Duplicate key "false" in map (foo: 1px, true: 4px).
        on line 1 of test.scss
>> $m: (foo: 1px, null: 2px, false: 3px, true: 4px);
```

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/437

**Comment by xzyfer** [comment]
This is fixed, and will be in 3.3. 

An additional aspect to this issue has been broken out into #1336
